src/widgets/PhotoGallery/PhotoGallery.js

```diff
@@ -0,0 +1,25 @@
+import './PhotoGallery.css';
+import { Counter } from '../../features/';
+import { Photos } from '../../features/';
+
```

Comment:
```suggestion
import { Counter } from 'features';
import { Photos } from 'features';
```

Внутри слоя пути относительные, а между слоями абсолютные